group/iab/IAB_Decisions.md

```diff
@@ -54,56 +59,47 @@ for the IAB chair to determine whether it will be held open or secret.
 ## Normal Process
 
 The IAB chair (or executive director on the instructions of the chair) will 
-typically follow the following process:
+typically follow this process to initiate a vote:
 
-- If the request for a vote comes from an external source (e.g., the NomCom
-  provides its IESG slate for confirmation), then, as soon as possible after
-  receiving the request, the IAB chair adds the vote topic to the agenda of the
-  next IAB teleconference. If the next meeting is less than 48 hours away, the
-  chair should instead add the topic to the following teleconference.
+- The chair emails the list of voting members to start an email discussion on the
+topic, define when the voting will begin, and optionally indicate that the ballots
+will be secret. In general, some reasonable amount of time (such as one week)
+should be allocated for discussion prior to the vote beginning, but the vote may
+begin without delay if necessary. In such cases, discussion happens in parallel
+with voting.
 
-- The chair emails the list of voting members to announce the date of the
-  discussion, whether ballots are open or secret, and start an email discussion
-  thread on the topic.
-  
-- The agenda of that meeting will include the vote discussion, and needs to be
-  posted at least 48 hours in advance. If the discussion will happen in an
-  irregularly scheduled meeting (i.e. meetings other than the weekly IAB
-  meeting teleconference or meetings prescheduled during IETF week) then, the
-  new meeting needs to be announced at least a week in advance.
+- The chair or a delegate (such as a secretariat member) will formally start a vote
+by emailing the list or as part of a teleconference or meeting. When a vote starts in
+a teleconference or meeting but does not record all members' positions, the vote will also be sent
+to the email list.
 
-- Discussion and votes will continue on the voting members list, and during the
-  teleconference.
+- Discussion and votes will continue on the voting members list, and during any
+  teleconference(s) that occur while the vote is open.
 
-- The vote will typically stay open until a vote has been recorded for every
+- The vote will typically stay open until a position has been recorded for every
   single voting member, even after a majority has been reached. This ensures
   every voice can be heard. Note that it is reasonable for any voting member to
   request a few days to review available information before voting.
 
```

Comment:
Why wouldn't we have an deadline date for a vote? Like 7 or 10 days or whatever. Otherwise, could one intransigent member prevent closure of the vote by not voting? ISTM we should have a "No Vote Received" option for this case.